.github/workflows/ci.yml

```diff
@@ -38,7 +38,7 @@ jobs:
         shell: bash
         # excludes tests, testdata, and generated sources from coverage report
         run: |
-         deno coverage ./cov/ --lcov --exclude="test\\.(ts|js)|wasm\\.js|testdata|node/_tools|node/_module/cjs|node_modules" > cov.lcov
+         deno coverage ./cov/ --lcov --exclude="test\\.(ts|js)|wasm\\.js|testdata|node/_tools|node/_module/cjs|node_modules|\\.snap" > cov.lcov
```

Comment:
While this change makes sense, I think we should also tackle https://github.com/denoland/deno/issues/14408 so it's not accounted for by default

Reply:
Yeah good call 😊 I will implement a fix for https://github.com/denoland/deno/issues/14408 and then remove this.

Reply:
This should now be addressed by https://github.com/denoland/deno/commit/253fbf9d2a15018cbba9c9ea2829bd70e9723362

Reply:
Thanks! :) I have reverted this change